src/Pages/Authentication/Login/Login.jsx

```diff
@@ -0,0 +1,95 @@
+                                placeholder='Enter email'
+                                value={loginData?.email? loginData.email : ''}
+                                onChange={(e)=>setLoginData(prevData => {return  {...prevData, email:e.target.value}})}
+                            />
+                             <InputField 
+                                type={showPassword?'text':'password'}
+                                title='Password'
+                                name='password'
+                                placeholder='Enter password'
+                                showPassword={showPassword}
+                                handlePassword = {handlePassword}
+                                value={loginData?.password? loginData.password : ''}
+                                onChange={(e)=>setLoginData(prevData => {return  {...prevData, password:e.target.value}})}
+                            />
+
+                            <div className="remember">
+                                <label>
+                                    <Input id='rememberUser' type="checkbox" name ='remember' checked={true}/>
+                                    Remember me!
+                                </label>
+                                <br/>
+                                <Link  to="/forgot-password" className="btn-link-forgot-password" target="_blank">Forgot Password ??</Link>
+                            </div>
+                            <button className="btn btn-sm btn-primary" type="submit" onClick={handleLogin}>Sign In</button>
+                            <div className="inputBx">
+                                {/* <p className="text-md">Don't have an account?<a className="btn text-decoration-none btn-sm btn-link"
+                                        href="../SignUp/SignUp.html">Sign Up</a></p> */}
+                                <p className="text-md">Don't have an account?
+                                    <Link  to="/signup" className="btn text-decoration-none btn-sm btn-link">Sign Up</Link>
+                                </p>
+                            </div>
+                        </form> 
+                        <div className="sm-container">
+                            <p className="text-lg">Login with social media:</p>
+                            <div className="flex-content">
+                                <img src="https://img.icons8.com/color/48/000000/facebook-circled--v1.png" className="icon-btn-img"
+                                    alt="icon-btn-img" />
+                                <img src="https://img.icons8.com/nolan/48/instagram-new.png" className="icon-btn-img"
+                                    alt="icon-btn-img" />
+                                <img src="https://img.icons8.com/fluency/48/000000/twitter.png" className="icon-btn-img"
+                                    alt="icon-btn-img" />
+                            </div>
+                        </div>
```

Comment:
Suggestion: I think you can create a diff component for this

Reply:
already have a different component for image